x-pack/test/api_integration/apis/monitoring/apm/index.js

```diff
@@ -8,10 +8,7 @@
 export default function ({ loadTestFile }) {
   describe('APM', () => {
     loadTestFile(require.resolve('./overview'));
-    loadTestFile(require.resolve('./overview_mb'));
```

Comment:
Replaced by the new test suites. this suite only run tests for legacy monitoring